packages/wrangler/src/init.ts

```diff
@@ -523,6 +525,8 @@ export async function initHandler(args: ArgumentsCamelCase<InitArgs>) {
 			);
 
 			if (fromDashScriptName) {
+				logger.warn(`After using "init --from-dash" it is suggested continued use of the CLI,
```

Comment:
Doesn't quite make sense:
"it is suggested to continue to use the CLI"
or "continued use of the CLI is suggested"

can we use the same message as above instead?